### ISSUE TITLE
How to ignore specific tokens?

### ISSUE THREAD

**Issue by alonsopg** [mammothb/symspellpy#17]
Given this string:

` 24th december`

symspell is correcting as:

In:
```
    new = sym_spell.word_segmentation('24th december')
    new[1]
```

Out: 

` beth december`

Is there any way of skipping those cases with a regular expression? That is all the tokens that match `\d{2}\bth\b` skip them? I tried to tokenize and apply the algorithm to each word. However, this damage the accuracy for short words.

**Comment by mammothb** [comment]
I made a commit to the `dev` branch which allows you to skip words which matches a certain regular expression. However, I was not able to skip the word `24th` with the expression you have provided. I was able to get the proper output with 
```
new = sym_spell.word_segmentation('24th december', ignore_token=r"\d{2}\w*\b")
```
I'm not that familiar with regular expression so maybe I am using the expression you provided incorrectly.



In your provided code, I assume your `max_edit_distance` > 0 in the instantiation of the `SymSpell` object. For the provided example, we can still get the proper output of `24th december` if we used
```
new = sym_spell.word_segmentation('24th december', max_edit_distance=0)
```
Just providing an alternative to consider since using `word_segmentation` with `max_edit_distance` > 0 can sometimes give worse results.
